inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -929,13 +933,20 @@ layout layout_optimizer::get_expected_layout(layout const& current_layout,
         } else if (input_layout.data_type == data_types::f16 &&
                 convolution_bs_fs_yx_bsv16_fsv16_opt(input_layout, output_layout, weights_layout, prim) &&
                 (output_layout.data_type == input_layout.data_type ||
-                !data_type_traits::is_floating_point(input_layout.data_type))) {
+                !data_type_traits::is_floating_point(input_layout.data_type)) && is_2d) {
             expected_tensor = current_layout.size;
             if (prim->groups == 1 || (output_layout.size.feature[0] % 16 == 0 && input_layout.size.feature[0] % 16 == 0)) {
                 expected_format = cldnn::format::bs_fs_yx_bsv32_fsv16;
             } else {
                 expected_format = cldnn::format::bs_fs_yx_bsv16_fsv16;
             }
+        // TODO: add this case when corresponding fsv32 optimizations inside cldnn will be implemented
+        } else if (input_layout.data_type == data_types::f32 && i8_u8_output && !is_first_conv && is_2d && false) {
```

Comment:
Why `&& false`? If it is not ready, what about removing or commenting out?

Reply:
I wouldn't like to delete this code, so I've commented it.